vod/manifest_utils.c

```diff
@@ -48,12 +48,34 @@ manifest_utils_write_bitmask(u_char* p, uint32_t bitmask, u_char letter)
 	return p;
 }
 
-static uint32_t*
+static u_char*
+manifest_utils_write_bitmask(u_char* p, uint64_t* bitmask, uint64_t* mask_temp, uint32_t max_bits, u_char letter)
```

Comment:
i think it's worth adding an intermediate function that is specific to track_mask_t, that will not require `mask_temp`